deployment-templates/forseti-instance.py

```diff
@@ -116,7 +116,7 @@ def GenerateConfig(context):
     if SHOULD_INVENTORY_GROUPS:
         GROUPS_DOMAIN_SUPER_ADMIN_EMAIL = context.properties[
             'groups-domain-super-admin-email']
-        GROUPS_SERVICE_ACCOUNT_KEY_FILE = context.properties[
+        GSUITE_SERVICE_ACCOUNT_KEY_FILE = context.properties[
```

Comment:
Actually, I change my mind about renaming this here.  Will do this separately in a follow-up PR, so that I can focus updating and testing everything this renaming will create.